src/Validation/src/ValidatablePropertyInfo.cs

```diff
@@ -59,6 +59,16 @@ protected ValidatablePropertyInfo(
 
     /// <inheritdoc />
     public virtual async Task ValidateAsync(object? value, ValidateContext context, CancellationToken cancellationToken)
+    {
+        await ValidatePropertyAttributesAsync(value, context, cancellationToken);
+        await ValidateComplexObjectsAsync(value, context, cancellationToken);
+    }
+
+    /// <summary>
+    /// Validates only the property attributes (Required, Range, etc.) without validating complex objects.
+    /// Returns true if there were validation errors.
+    /// </summary>
+    public virtual Task<bool> ValidatePropertyAttributesAsync(object? value, ValidateContext context, CancellationToken cancellationToken)
```

Comment:
The new `ValidatePropertyAttributesAsync` and `ValidateComplexObjectsAsync` methods do not need to be `public`.  Make them internal and remove them from PublicApi.Unshipped.txt.